src/app/shared/faq-assistant/faq-assistant.component.scss

```diff
@@ -1,15 +1,18 @@
+body, li {
```

Comment:
Components shouldn't change body and other tags because this will affect entire site.

Reply:
faq-assistant